tests/testthat/test-project_as_ppi.R

```diff
@@ -1,5 +1,15 @@
 test_that("returns error on incorrect parameters", {
-
+  # I don't think a user could naturally reach these errors:
+  expect_error(
+    project_as_ppi.param(example_scan),
+    regexp = 'inherits(x, "param") is not TRUE',
+    fixed = TRUE
+  )
+  expect_error(
+    project_as_ppi.scan(get_param(example_scan, "DBZH")),
+    regexp = 'inherits(x, "scan") is not TRUE',
+    fixed = TRUE
+  )
```

Comment:
While these error states were uncovered, I do not believe a user could actually trigger these errors naturally. Either they pass a `scan` or a `param` object, or something else entirely. If they pass a `scan` or `param` object, no error should be returned, if they pass anything else, they should get; 

```
Error in UseMethod("project_as_ppi", x) : 
  no applicable method for 'project_as_ppi' applied to an object of class "{class_of_invalid_input_object}"
```

Reply:
true, should test only against `project_as_ppi()` and not its class methods. We could strip away the .param and .scan postfix to the main function